backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Fresh_PriceList_Details_ToDate_Report.sql

```diff
@@ -68,17 +76,29 @@ BEGIN
     CREATE TEMP TABLE IF NOT EXISTS temp_price_list_details AS
     SELECT DISTINCT bp.c_bpartner_id,
                     plv.M_PriceList_Version_ID,
-                    plv.name
+                    plv.name,
+                    (SELECT bpl.c_bpartner_location_id
+                     FROM c_bpartner_location bpl
+                     WHERE bpl.c_bpartner_id = bp.c_bpartner_id
+                     ORDER BY bpl.isbilltodefault
+                             DESC
+                     LIMIT 1) AS c_bpartner_location_id,
+                    bp.ad_org_id
     FROM c_bpartner bp
              INNER JOIN M_PriceList pl ON bp.m_pricingsystem_id = pl.m_pricingsystem_id
              INNER JOIN M_PriceList_Version plv ON pl.m_pricelist_id = plv.m_pricelist_id
     WHERE bp.iscustomer = 'Y'
```

Comment:
Sincenow we have a parameter that filter for sales or purchase, I do no think we shall filter the partner with flag ```IsCustomer='Y'````

Reply:
done